tests/test.env.example

```diff
@@ -1,3 +1,3 @@
 export KB_DEPLOYMENT_CONFIG_DIR=/Users/<username>/<some_dir>/staging_service/deployment/conf
 export KB_DEPLOYMENT_CONFIG=${KB_DEPLOYMENT_CONFIG_DIR}/testing.cfg
-export MAPPINGS_FILE=${KB_DEPLOYMENT_CONFIG_DIR}/supported_apps_w_extensions.json
\ No newline at end of file
+export FILE_EXTENSION_MAPPINGS=${KB_DEPLOYMENT_CONFIG_DIR}/supported_apps_w_extensions.json
```

Comment:
Seems to be a typo. `MAPPINGS_FILE` doesn't appear anywhere else in the codebase.